libs/generators/src/lib/generators/generators-menu/generators-menu.component.scss

```diff
@@ -1,6 +1,7 @@
 :host {
-    .menu-item {
-        justify-content: left;
-        width: var(--menu-item-width);
-    }
-}
\ No newline at end of file
+  .menu-item {
+    justify-content: left;
+    text-transform: capitalize;
```

Comment:
I would prefer to use an object with title and path properties instead of having only one and manipulating it with css

Reply:
@ronnetzer I agree, but this is just a mock, we will get the data of the available generators from the backend.

Reply:
@eladzipper96 I'm pretty sure we are not, this menu toggles between types of generators which are constants